packages/quantic/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle.css

```diff
@@ -28,7 +28,11 @@
 .searchbox__submit-button {
   width: 46.75px;
   border-radius: 0;
-  height: 100%;
+  height: 48px;
+}
+
+.searchbox__clear-button {
+  height: 48px;
```

Comment:
```suggestion
  height: 3rem;
```

Reply:
Nope, the other heights are all set to exactly 48px so it needs to be consistent. Don't know where this value comes from but it is what it is.